src/functions.php

```diff
@@ -214,18 +243,23 @@ function dissolveGroups(array $vcards): array
 /**
  * Filter included/excluded vcards
  *
- * @param stdClass[] $vcards
+ * @param mixed[] $vcards
  * @param array $filters
- * @return stdClass[]
+ * @return mixed[]
  */
 function filter(array $vcards, array $filters): array
 {
+    $vcf = '';
+    foreach ($vcards as $vcard) {
+        $vcf = $vcf . $vcard->serialize();
+    }
+    file_put_contents('TEST.txt', $vcf);
+
     // include selected
     $includeFilter = $filters['include'] ?? [];
 
-    if (countFilters($includeFilter)) {
+    if (countFilters($includeFilter) > 0) {
```

Comment:
This was a bug

Reply:
Ich versteh nicht? Ohne >0 gings nicht? Kann ich mir nicht vorstellen.

Reply:
counterfilters liefert immer true! Auch wenn es null - also keine Definitionen enthält! Dann ist es aber nicht unsinnig die Bedingung zu durchlaufen, weil ja (keine Includedefinition = alles includen). Die IF bedingung (dezidierte Prüfung) zu durchlaufen ist nur notwendig wenn >= 1 Filterdefinition vorliegt.

Reply:
Siehe https://3v4l.org/dHe3B das ist NICHT nötig. Ich wiederhole mich da. Du behauptest es ist nötig. NEIN.